common/src/jni/main/cpp/NativeCrypto.cpp

```diff
@@ -4378,6 +4378,29 @@ static jlong NativeCrypto_asn1_read_sequence(JNIEnv* env, jclass, jlong cbsRef)
     return reinterpret_cast<uintptr_t>(seq.release());
 }
 
+static jboolean NativeCrypto_asn1_read_next_tag_is(CONSCRYPT_UNUSED JNIEnv* env, jclass, jlong cbsRef, jint tag) {
```

Comment:
why not _asn1_peek_next_tag to match boringssl naming ? not a big deal either way though, since the names aren't going to be an exact match.